docs/sidebars.js

```diff
@@ -1,308 +1,321 @@
-                    label: 'Preparing For Production',
-                    collapsed: true,
-                    items: [
-                        'messaging-and-voice-channels',
-                        'tuning-your-model',
-                        'testing-your-assistant',
-                        'setting-up-ci-cd',
-                    ],
-                },
-                'glossary',
-            ],
+          type: "category",
+          label: "Actions",
+          items: [
+            "building-assistants/actions",
+            "building-assistants/custom-actions",
+            "building-assistants/default-actions",
+            "building-assistants/slot-validation-actions",
+          ],
         },
+        "building-assistants/llm-command-generator",
+        "building-assistants/responses",
         {
-            type: 'category',
-            label: 'Deploying Assistants',
-            collapsed: true,
-            items: [
-                'deploy/introduction',
-                'deploy/deploy-rasa',
-                'deploy/deploy-action-server',
-                'deploy/deploy-rasa-pro-services',
-            ],
+          type: "category",
+          label: "Config",
+          items: [
+            "building-assistants/llm-configuration",
+            "building-assistants/custom-graph-components",
+            "building-assistants/training-data-importers",
+            "building-assistants/graph-recipe",
+            "building-assistants/spaces",
+          ],
         },
+        "building-assistants/policies", // TODO: create-task
+        // TODO: create-task spit CLI docs
+        "building-assistants/command-line-interface",
+        // TODO: create-task add how-to-guides
+        "building-assistants/glossary",
         {
-            type: 'category',
-            label: 'Monitoring and Analyzing Assistants',
-            collapsed: true,
-            items: [
-                {
-                    type: 'category',
-                    label: 'Analytics',
-                    collapsed: true,
-                    items: [
-                        'monitoring/analytics/getting-started-with-analytics',
-                        'monitoring/analytics/realtime-markers',
-                        'monitoring/analytics/example-queries',
-                        'monitoring/analytics/data-structure-reference',
-                    ],
-                },
-                'monitoring/tracing',
-                'monitoring/load-testing-guidelines',
-            ],
+          type: "category",
+          label: "Building Classic Assistants",
+          collapsed: true,
+          items: [
+            // TODO: we'll need something for DM2 at some point
+            "building-classic-assistants/migrate-from",
+            // TODO: create-task spit CLI docs
+            "building-assistants/command-line-interface",
+            {
+              type: "category",
+              label: "Conversation Patterns",
```

Comment:
how about we make a banner similar to the Rasa Pro one for all "classic" pages? in case you land there via search

Reply:
yes, will create a task